clients/drcachesim/tools/view.h

```diff
@@ -79,6 +79,14 @@ class view_t : public analysis_tool_t {
     bool
     should_skip(const memref_t &memref);
 
+    inline void
+    print_header()
```

Comment:
The description says you want to override this in subclasses: so it needs to be virtual.

Reply:
Not override, just invoke. E.g. if the sub-class overrides the (virtual) initialize routine, it will invoke this print_header routine by itself.